### PR TITLE
Add error handling to sound object

### PR DESCRIPTION
**Pull request by kylesuss**
Some error handling would be nice :)

### CODE REVIEW

**Review by peinguin — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -154,6 +156,9 @@ export default class Sound extends React.Component {
       whileplaying() {
         props.onPlaying(this);
       },
+      onerror() {
```
Comment (leoasis) [inline src/index.js]: This event receives some useful parameters, such as the error number 
 and description (http://www.schillmania.com/projects/soundmanager2/doc/#smsound-onerror). Maybe we could add them as parameters in the callback prop? we could do:
```js
props.onError(errorCode, description, this)
```

File: README.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ class MyComponentWithSound extends React.Component {
 * *volume (number)*: The current sound's volume. A value between 0 and 100.
 * *autoLoad (boolean)*: If the sound should start loading automatically (defaults to `false`).
 * *loop (boolean)*: If the sound should continue playing in a loop (defaults to `false`).
+* *onError (function)*: Function that gets called when the sound fails to load, or fails during load or playback.
```
Comment (leoasis) [inline README.md]: We should explain that is received in this function

**Review by leoasis — CHANGES_REQUESTED** [review]
Thanks for the contribution! In fact this is indeed necessary, glad you took the work to add it. There is a comment however that we need to address before merging.

File: src/index.js (hunk 1 above)
Comment (kylesuss) [reply src/index.js]: Sweet. Sounds good. I'll get around to this later on today, then resubmit 👌

**Comment by kylesuss** [comment]
@leoasis Ok I made the updates!

**Review by leoasis — APPROVED** [review]
Thank you for the contribution! this looks great

**Pull request merged**